operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```
@@ -597,7 +597,16 @@ local utils = import 'mixin-utils/utils.libsonnet';
     )
     .addPanel(
       $.panel('Inflight requests') +
-      $.queryPanel('sum(cortex_bucket_stores_gate_queries_in_flight{%s, component="%s"})' % [$.namespaceMatcher(), component], 'Total')
+      $.queryPanel(
+        [
+          'sum(cortex_bucket_stores_gate_queries_in_flight{%s, component="%s"})' % [$.namespaceMatcher(), component],
+          'max(cortex_bucket_stores_gate_queries_in_flight{%s, component="%s"})' % [$.namespaceMatcher(), component],
```

```suggestion
          'max(max_over_time(cortex_bucket_stores_gate_queries_in_flight{%s, component="%s"}[$__interval])' % [$.namespaceMatcher(), component],
```

Using `max_over_time` means you'll be able see more peaks as it will take the max of all the samples in the interval, whereas using just `max` means you're only looking at the last sample in the interval.